js/components/controlBar.js

```diff
@@ -478,6 +486,12 @@ var ControlBar = React.createClass({
       "oo-selected": this.props.controller.state.closedCaptionOptions.showPopover
     });
 
+    var multiAudioClass = ClassNames({
+      "oo-multiaudio": true,
+      "oo-control-bar-item": true,
+      "oo-selected": this.props.controller.state.multiAudioOptions.showPopover
```

Comment:
Is this state defined yet? Or will this be defined in a future commit? I don't see ``this.props.controller.state.multiAudioOptions`` defined on the master branch, so this will throw a null pointer exception if this is merged as is.